src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md

```diff
@@ -53,6 +46,118 @@ On Windows (Powershell)
+    <pluginGroups>
+        <pluginGroup>dev.galasa</pluginGroup>
+    </pluginGroups>
+     
+    <profiles>
+        <profile>
+            <id>galasa</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <repositories>
+                <repository>
+                    <id>maven.central</id>
+                    <url>https://repo.maven.apache.org/maven2/</url>
+                </repository>
+                <!-- To use the bleeding edge version of galasa, use the development obr
+                <repository>
+                    <id>galasa.repo</id>
+                    <url>https://development.galasa.dev/main/maven-repo/obr</url> 
+                </repository>
+                -->
+            </repositories>
+            <pluginRepositories>
+                <pluginRepository>
+                    <id>maven.central</id>
+                    <url>https://repo.maven.apache.org/maven2/</url>
+                </pluginRepository>
+                <!-- To use the bleeding edge version of galasa, use the development obr
+                <pluginRepository>
+                    <id>galasa.repo</id>    
+                    <url>https://development.galasa.dev/main/maven-repo/obr</url> 
+                </pluginRepository>
+                -->
+             </pluginRepositories>
+         </profile>
+     </profiles>
+</settings>
+```
+This information tells Maven where to find some custom Maven plug-in tools that are used by the build.
+
+
+## Installing Gradle
+
+1. Install <a href="https://gradle.org/install/" target="_blank"> Gradle</a> version 6.x. *Note:* Gradle version 7.x is not currently supported.
+2. Put Gradle on your PATH by adding the following information to your shell’s initialization file:
+```
+export PATH="/opt/homebrew/opt/gradle@6/bin:$PATH"
+gradle --version | grep "Gradle" | cut -f2 -d' '
+```
+
+## About Galasa CLI commands
+
+Galasa CLI commands start with `galasactl`. Example commands are provided for running on Mac or Unix, and Windows Powershell. The Windows Powershell uses the backtick (`) for line continuation characters. If you are using Windows command-shell, the line continuation character is the caret (^). 
+
+You can view the Galasa CLI command syntax, including parameter descriptions, in the <a href=https://github.com/galasa-dev/cli/blob/main/docs/generated/galasactl.md target="_blank"> cli repository</a> in GitHub.
+
+
+## Known limitations
+
+Go programs can sometimes struggle to resolve DNS names, especially when a working over a virtual private network (VPN). In such situations, you might notice that a bootstrap file cannot be found with galasactl, but can be found by a desktop browser, or curl command. In such situations you can manually add the host detail to the `/etc/hosts` file, to avoid DNS being involved in the resolution mechanism.
```

Comment:
What is /etc/hosts on a windows machine ?

Reply:
it is here c:\Windows\System32\Drivers\etc\hosts, so will update to be \etc\hosts